corehq/apps/ota/utils.py

```diff
@@ -212,4 +213,4 @@ def update_device_id(user, device_id):
         if not user.is_demo_user:
             updated = user.update_device_id_last_used(device_id)
             if updated:
-                user.save(fire_signals=False)
+                user.save(fire_signals=toggles.ENIKSHAY.enabled(user.domain))
```

Comment:
will this toggle get queried every time any user syncs now? (I know `enikshay` is an "always on" domain for this, but on other envs / projects).

Reply:
That's true - it will get called for every sync where the device id isn't the same as the previous one for that user.  I've generally been treating `toggles.X.enabled` as a ~free lookup, but if that's not the case, it should be.